README.md

```diff
@@ -1,86 +1,36 @@
+Please refer to the [Steward Docs](docs/) for setup instructions.
 
-### Testing mode
-
-These instructions assume that the Cellar has been deployed to the target ethereum blockchain.
-
-#### Setup Configuration File
-
-The first step is to setup your configuration file.
-
-To generate a configuation file template, run the command below in your terminal:
-
-```
-cargo run --bin steward print-config
-```
-
-Next, create a `toml` file in the root of the application, replacing the default keys in the template displayed in your terminal with your configuration. Make sure to confirm that the token info in your configuration file matches the deployed Cellar contract.
-
-You can create keys or import keys. To create keys, run the command below. Replace `eth` with `cosmos` if you want to create a `cosmos` key instead of an `eth` key.
-
-```
-cargo run --bin steward -- -c [your_config_file_name.toml] keys eth add [key_name]
-```
-
-If you already have a key, you can import it with the command below:
+## Steward use case
 
-```
-cargo run --bin steward -- -c [your_config_file_name.toml] keys eth import [key_name]
-```
+Steward wears several hats:
 
-Now, navigate to the keystore location in your local environment, i.e `keys.keystore`. Confirm that the key was created successfully in the location you specified in your config file.
+###  1. Gravity Bridge Operator
 
-#### Authorize Erc20 Token to interact with Cellar contract.
+Steward is responsible for running the Relayer, which handles relaying Cosmos transactions to Ethereum, and the Orchestrator, which enables the co-processing of Ethereum transactions on Sommelier. Steward runs the Orchestrator so that Sommelier can manage Cellars on Ethereum. Which brings us to Steward's second responsibility.
 
-Run the `allow-erc-2-0` command as shown below, to allow Erc20 token to interact with Cellar contract.
+### 2. Middleman between the Sommelier chain and Strategy Providers
 
-```
-cargo run --bin steward -- -c [your_config_file_name.toml] allow-erc-2-0 --cellar-address=[the_cellar_address] address [the_erc20_address] --amount [amount]
+Steward runs on every Validator in the Sommelier Validator set. It runs a server to which Strategy Providers (SPs) send requests whenever they determine that the market has changed enough to warrant action. The request payload contains everything needed to make a *cork*: a signed combination of a cellar address and an ABI encoded contract call. When Steward receives a submission from the SP, it validates the target cellar address, builds a cork, signs it with the delegate key, and submits it to the [Cork module](https://github.com/PeggyJV/sommelier/tree/main/x/cork) on chain, where it will be evaluated before being bridged to the target Cellar contract.
```

Comment:
"warrant action" is very vague here, should probably indicate this is talking about making a call to a cellar to update it.

Reply:
In the last sentence, "where it will be evaluated...", might be more appropriate to say where it will be bridged to the target cellar contract if a consensus of validators submit the same cork.